atomist/get-started.md

```diff
@@ -0,0 +1,436 @@
+---
+title: Get started
+description: Getting started with Atomist
+keywords: atomist, software supply chain, vulnerability scanning, tutorial
+toc_max: 2
+---
+
+{% include atomist/disclaimer.md %}
+
+To get started with Atomist, you'll need to:
+
+- Connect Atomist with your container registry
+- Link your container images with their Git source
+
+Before you can begin the setup, you’ll need a Docker ID. If you don't already
+have one, you can [register here](https://hub.docker.com/signup){:
+target="blank" rel="noopener" class=""}.
+
+## Connect container registry
+
+After completing this setup, Atomist will have read-only access to your
```

Comment:
I'd move this sentence to the top and make it part of the relevant bullet point. Having it here confused me for a second, it felt like I'd skipped something.

Reply:
Good point... I tried to change the introduction text for this section to clarify. I think it could be good to have this information up-front, about what access Atomist requires and why.